index.bs

```diff
@@ -1926,7 +1926,7 @@ follows.
 
 The <dfn>client data</dfn> represents the contextual bindings of both the [=[RP]=] and the client platform. It is a key-value
 mapping with string-valued keys. Values can be any type that has a valid encoding in JSON. Its structure is defined by the
-following Web IDL.
+following Web IDL but it's important to note that it may be extended in the future. Therefore it's critical when parsing to be tolerant of unknown keys and of any reordering of the keys.
```

Comment:
suggest making the new text a "Note:"

IIUC, we may want to add that the [=CTAP2 canonical CBOR encoding form=] stipulates sorted keys thus adding other keys will likely alter ordering one might otherwise be expecting.

Reply:
The client data doesn't contain any CBOR data, though. :)

Reply:
doh  0_o

Reply:
Done.